src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/StrandBiasUtils.java

```diff
@@ -38,21 +38,37 @@
      * @param perAlleleValues forward and reverse read counts for each allele
      * @return a String appropriate to use for annotating a GVCF
      */
-    protected static String makeRawAnnotationString(final List<Allele> vcAlleles, final Map<Allele, List<Integer>> perAlleleValues) {
-        final List<String> alleleStrings = vcAlleles.stream()
-                // does not replace a null value with zero list - only if the key is not in the map
-                .map(a -> perAlleleValues.getOrDefault(a, ZERO_LIST))
-                .map(StrandBiasUtils::encode)
-                .collect(Collectors.toList());
-        return String.join(AnnotationUtils.ALLELE_SPECIFIC_RAW_DELIM, alleleStrings);
-
+    public static String makeRawAnnotationString(final List<Allele> vcAlleles, final Map<Allele, List<Integer>> perAlleleValues) {
```

Comment:
Since you are reverting this method back to a version that handles nulls, I think you should remove the `perAlleleValues.values().removeIf(Objects::isNull);` line you added to `computeSBAnnotation()` above as well, for consistency with the pre-bug behavior.

Reply:
reverted to previous code